packages/theme/src/color-ramps/test/index.test.ts

```diff
@@ -0,0 +1,94 @@
+/**
+ * External dependencies
+ */
+// Not sure why ESLint is erroring.
+// eslint-disable-next-line import/no-extraneous-dependencies
+import Color from 'colorjs.io';
+
+/**
+ * Internal dependencies
+ */
+import { buildRamp } from '../lib';
+import { BG_RAMP_CONFIG, ACCENT_RAMP_CONFIG } from '../lib/ramp-configs';
+import { DEFAULT_SEED_COLORS } from '../lib/constants';
+
+describe( 'buildRamps', () => {
```

Comment:
I greatly reduced the number of combinations run in both of these snapshot tests (98ab6ed132075fdb62d198fba2e120df40b65dd3), because it was taking 13 secs to run, and resulted in a snapshot file of more than 1MB. Now it is down to 450ms and 30kb.

I'm actually not sure if the original number of combinations were meaningfully adding to our confidence levels, but if so, we should consider keeping such a costly test skipped so it can be used by a maintainer only when needed.